.github/workflows/profiling.yaml

```diff
@@ -49,8 +41,13 @@ jobs:
           cat ${GITHUB_EVENT_PATH}
 
           echo "Creating Download URL from ISSUE"
-          echo "DOWNLOAD_URL=$(cat ${GITHUB_EVENT_PATH} | jq '.comment.body' | grep -oP '(https:)(.*?)(?=\))' )" >> ${GITHUB_ENV}
-
+          if ! echo "DOWNLOAD_URL=$(cat ${GITHUB_EVENT_PATH} | jq '.comment.body' | grep -oP '(https:)(.*?)(?=\))' )" >> ${GITHUB_ENV}; then
+            echo "Must be an issue body"
+          else
```

Comment:
Should we maybe also _skip_ instead of failing when there is no download URL to be found?

Reply:
Makes sense. I think we can leave it as it is then, unless it makes noise on PRs for which performance tests should _not_ be run at all.